www-src/vendor/scripts/moment.js

```diff
@@ -0,0 +1 @@
+../../node_modules/moment-timezone/node_modules/moment/moment.js
```

Comment:
Wow, ok ^^.
I will put the trick with moment as a project dependency, to have a consistent solution for npm2 and npm3 .